### DISCUSSION TITLE
Audio over xrdp to Remote Desktop Connection

### DISCUSSION THREAD

**Discussion by bowmanpete123** [microsoft/WSL#6142] [General]
**Is your feature request related to a problem? Please describe.**
Not really, as there are many fixes for this using Xserver or other connection programs

**Describe the solution you'd like**
I'd like WSL to support audio through a connection with xrdp on one side and windows remote desktop on the other 

**Describe alternatives you've considered**
Right now i'm running audio/video through the Xserver, but I prefer using the rdp thats built in to windows

**Additional context**

**Comment by therealkenc** [comment]
You will get better eyeballs on this question in an [xrdp](https://github.com/neutrinolabs/xrdp) forum. There is a MSFT RDP client Q&A forum [here](https://docs.microsoft.com/en-us/answers/topics/windows-remote-desktop-client.html).

GUI apps on WSL is #938. Audio on WSL2 is #5816.
